.markdownlint.json

```diff
@@ -5,8 +5,8 @@
         "line_length": 250
     },
     "list-marker-space": {
-        "ol_single": 3,
-        "ol_multi": 3,
+        "ol_single": 2,
```

Comment:
Once again: please don't mix changes. For now it's OK to leave these changes in, but try not to make this again in the future.

This makes the diff unnecessarily large, and it's also a sign of loss of focus. If you're working on the CLI there's no good reason to tweak these values.

Reply:
I also noticed that you made a bunch of changes to the docs, which I actually really appreciate :)

But again: focus focus focus. It's all about focus! We'll work on improving the docs later.

Reply:
Makes sense, but in this case, the changes were necessary for updating the documentation.